src/main/java/net/syscon/elite/api/model/CourtHearing.java

```diff
@@ -22,12 +21,9 @@
     @ApiModelProperty(value = "The court hearing identifier.", position = 1, example = "123456789")
     private Long id;
 
-    @ApiModelProperty(value = "The date of the court hearing.", position = 2, example = "2020-03-01")
-    private LocalDate date;
+    @ApiModelProperty(value = "The date and start time of the court hearing.", position = 2, example = "2020-02-28T14:40:00.000Z")
+    private LocalDateTime dateTime;
```

Comment:
This was originally based on the NOMIS db table structure but no real need to be.